scripts/gen.sh

```diff
@@ -7,8 +7,8 @@ readonly script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 cd "$script_dir/.."
 
-kdrfc -h3 draft-normington-jsonpath-latest.md && \
-xmlstarlet sel -T -t -v '//artwork[@type="abnf"]' ./draft-normington-jsonpath-latest.xml 2>/dev/null \
+kdrfc -h3 draft-normington-jsonpath-00.md && \
+xmlstarlet sel -T -t -v '//artwork[@type="abnf"]' ./draft-normington-jsonpath-00.xml 2>/dev/null \
 | aex \
-| bap -S path -q && cp draft-normington-jsonpath-latest.html docs/index.html
+| bap -S path -q && cp draft-normington-jsonpath-00.html docs/index.html
 )
```

Comment:
This needs reworking to match the new naming scheme `draft-normington-jsonpath.*`.

Reply:
Done in e28d965

Reply:
Thanks. For the record, I marginally prefer the explicit `-latest` suffix as it looks like a placeholder waiting to be replaced by a version number. But I can certainly live with this alternative.